fluttr_app/explorer_app/android/app/src/main/res/drawable/launch_background.xml

```diff
@@ -1,12 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Modify this file to customize your launch splash screen -->
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:drawable="@android:color/white" />
+    <item android:drawable="@color/grey" />
```

Comment:
Why did you choose grey here? Imo white would have looked better 😊

Reply:
I think a grey color theme fits to the blue of our logo.